### ISSUE TITLE
Viewer can be null in meeting selector 

### ISSUE THREAD

**Issue by nickoferrall** [ParabolInc/parabol#4614]
Sentry reports that the viewer in the meeting selector can be null causing the app to crash. 

This bug has affected 23 users over the past 2 months.

https://sentry.io/organizations/parabol/issues/2085248583/?referrer=github_plugin

```
TypeError: Cannot destructure property 'isConnected' of 'n' as it is null.
  at c (./packages/client/components/MeetingSelector.tsx:31:10)
  at Array.forEach (<anonymous>)
  at Set.forEach (<anonymous>)
...
(35 additional frame(s) were not displayed)
```

**Comment by jordanh** [comment]
I _love_ you're putting the number of active users on these. Thank you!

**Comment by jordanh** [comment]
Watermark: 109 users affected

**Comment by tiffanyhan** [comment]
Sentry now reports 5K occurrences across 5K users:

https://sentry.io/organizations/parabol/issues/2514125484/?project=107196&query=+level%3Afatal+timesSeen%3A%3E5+is%3Aunassigned&statsPeriod=14d

Related error bc it's just one line done from the previous error. ~900 occurrences across 900 users. props cannot be destructed:

https://sentry.io/organizations/parabol/issues/2514124635/?project=107196&query=+level%3Afatal+timesSeen%3A%3E5+is%3Aunassigned&statsPeriod=14d

**Comment by tiffanyhan** [comment]
related #5282

**Comment by mattkrick** [comment]
Cautiously optimistic that this is fixed in 6.25.0!

Last seen in 6.23.1: https://sentry.io/organizations/parabol/issues/2514125484/?project=107196&query=is%3Aunresolved+Cannot+destructure+property+%27isConnected%27&statsPeriod=14d

Gonna close & :pray: that it stays closed :crossed_fingers:
